R/unlockREDCap.R

```diff
@@ -278,9 +131,8 @@ connectAndCheck <- function(key, url, ...)
 #'          which returns the keys as a list. Use [globalenv()] to assign in the
 #'          global environment. Will accept a number such a '1' for global as well.
 #' @param keyring character. Potential keyring, not used by default.
```

Comment:
I think this is required - fix may be needed in "shelter::unlockKeys" documentation too